terraform/modules/aws/rout53/main.tf

```diff
@@ -0,0 +1,16 @@
+provider "aws" {
+  region = "eu-north-1"
+}
+
+resource "aws_route53_zone" "zone" {
+  name = var.domain_name
+}
+
+resource "aws_route53_record" "record" {
+  count   = length(var.records)
+  zone_id = aws_route53_zone.zone.zone_id
+  name    = var.records[0].name
```

Comment:
hmm so we will only create records with name equal to first one in the list :thinking: ?